client/src/components/Contact/ContactUs.jsx

```diff
@@ -0,0 +1,135 @@
+import { useRef, useState } from "react";
+import emailjs from "@emailjs/browser";
+import "./ContactPage.css";
+
+function ContactUs() {
+  const form = useRef();
+
+  const [showConfirmation, setShowConfirmation] = useState(false);
+
+  const [formName, setFormName] = useState("");
+
+  const [formMail, setFormMail] = useState("");
+
+  const [formMsg, setFormMsg] = useState("");
+
+  const displayConfirmation = () => {
+    if (formName !== "" || formMail !== "" || formMsg !== "") {
+      setShowConfirmation(true);
+      setFormName("");
+      setFormMail("");
+      setFormMsg("");
+    } else {
+      setShowConfirmation(false);
+    }
+  };
+  const handleChangeName = (e) => {
+    setFormName(e.target.value);
+  };
+
+  const handleChangeMail = (e) => {
+    setFormMail(e.target.value);
+  };
+
+  const handleChangeMsg = (e) => {
+    setFormMsg(e.target.value);
+  };
+  const sendEmail = (e) => {
+    e.preventDefault();
+
+    emailjs
+      .sendForm("service_2b3kg3s", "template_idjiy8i", form.current, {
+        publicKey: "7aPbsdkTmDMMrOxsJ",
+      })
+      .then(
+        () => {},
```

Comment:
?

Reply:
this is the form to send emails to customers

Reply:
J'ai supprimer le reste qui sert a rien
 .then(
        () => {},
        () => {}
      );